src/TestUtils/src/Microsoft.Maui.IntegrationTests/BaseBuildTest.cs

```diff
@@ -6,6 +6,9 @@ public class BaseBuildTest
 		public const string DotNetCurrent = "net8.0";
 		public const string DotNetPrevious = "net7.0";
 
+		public const string MauiVersionCurrent = "8.0.0-rc.1.9171"; // this should not be the same as the last release
```

Comment:
This comment is confusing. Do you mean it should not be the same as `MauiVersionPrevious`?

Reply:
Yeah, that was confusing. I'll fix.